extensions/wikia/FounderEmails/events/FounderEmailsEditEvent.class.php

```diff
@@ -43,151 +147,121 @@ public function enabled ( $wgCityId, User $user ) {
+	private function updateUserNotificationCount( User $user ) {
+		$counter = $this->getNotificationCounter( $user );
+		$counter[F::app()->wg->CityId]['count']++;
 
-				if ( !empty( $aWikiCounter[0] ) && $aWikiCounter[0] == $today && $aWikiCounter[1] === 'full' ) {
-					wfProfileOut( __METHOD__ );
-					return true;
-				}
+		$user->setOption( 'founderemails-counter', serialize( $counter ) );
+		$user->saveSettings();
+	}
 
-				// initialize or reset counter for today
-				if ( empty( $aWikiCounter[0] ) || $aWikiCounter[0] !== $today ) {
-					$aWikiCounter[0] = $today;
-					$aWikiCounter[1] = 0;
-				}
-				self::addParamsUser( $wgCityId, $user->getName(), $emailParams );
-				$mailCategory = FounderEmailsEvent::CATEGORY_DEFAULT;
-				// @FIXME magic number, move to config
-				if ( $aWikiCounter[1] === 15 ) {
-					$msgKeys['subject'] = 'founderemails-lot-happening-subject';
-					$msgKeys['body'] = 'founderemails-lot-happening-body';
-					$msgKeys['body-html'] = 'founderemails-lot-happening-body-HTML';
-					$mailCategory = FounderEmailsEvent::CATEGORY_EDIT_HIGH_ACTIVITY;
-					$mailKey = 'lot-happening';
-				} elseif ( $eventData['data']['registeredUserFirstEdit'] ) {
-					$msgKeys['subject'] = 'founderemails' . $wikiType . '-email-page-edited-reg-user-first-edit-subject';
-					$msgKeys['body'] = 'founderemails' . $wikiType . '-email-page-edited-reg-user-first-edit-body';
-					$msgKeys['body-html'] = 'founderemails' . $wikiType . '-email-page-edited-reg-user-first-edit-body-HTML';
-					$mailCategory = FounderEmailsEvent::CATEGORY_FIRST_EDIT_USER;
-					$mailKey = 'first-edit';
-				} elseif ( $eventData['data']['registeredUser'] ) {
-					$msgKeys['subject'] = 'founderemails' . $wikiType . '-email-page-edited-reg-user-subject';
-					$msgKeys['body'] = 'founderemails' . $wikiType . '-email-page-edited-reg-user-body';
-					$msgKeys['body-html'] = 'founderemails' . $wikiType . '-email-page-edited-reg-user-body-HTML';
-					$mailCategory = FounderEmailsEvent::CATEGORY_EDIT_USER;
-					$mailKey = 'general-edit';
-				} else {
-					$msgKeys['subject'] = 'founderemails' . $wikiType . '-email-page-edited-anon-subject';
-					$msgKeys['body'] = 'founderemails' . $wikiType . '-email-page-edited-anon-body';
-					$msgKeys['body-html'] = 'founderemails' . $wikiType . '-email-page-edited-anon-body-HTML';
-					$mailCategory = FounderEmailsEvent::CATEGORY_EDIT_ANON;
-					$mailKey = 'anon-edit';
-				}
+	private function sendUsingEmailExtension( $controller, User $user, array $eventData ) {
+		$params = [
+			'targetUser' => $user->getName(),
+			'currentUser' => $eventData['editorName'],
+			'pageTitle' => $eventData['titleText'],
+			'pageNs' => $eventData['titleNs'],
+			'previousRevId' => $eventData['previousRevId'],
+			'currentRevId' => $eventData['currentRevId'],
+			'fromAddress' => \F::app()->wg->PasswordSender,
+			'fromName' => \F::app()->wg->PasswordSenderName,
```

Comment:
Yah, I remembered this then looked around in vain to try to find where you had done this.  I just gave up figuring you'd point this out when the time came ;)